_episodes_rmd/60-next-steps.Rmd

```diff
@@ -29,7 +29,7 @@ data they need to handle.
 
 This aspect is central to the **Bioconductor**[^Bioconductor] project which
 uses the same **core data infrastructure** across packages. This certainly contributed
-to Bioconductor's success. Bioconductor package developers are advised to make
+to Bioconductors success. Bioconductor package developers are advised to make
```

Comment:
This should actually be the possive form - thus Bioconductor's success is correct, I believe.

Reply:
Resolved it by reverting changes.